doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -827,7 +795,7 @@ name "/F240/"
 
 \end_inset
 
- Any information found in the 
+ Beagle does not remove any information of the provided 
```

Comment:
“to remove from”!

“Beagle does not remove any information from the provided […]”